lib/packs/rails/railtie.rb

```diff
@@ -11,7 +11,15 @@ class Railtie < ::Rails::Railtie
         # hook into packs-rails via ActiveSupport hooks.
         ActiveSupport.run_load_hooks(:packs_rails, Packs)
       end
-      
+
+      config.after_initialize do |app|
```

Comment:
Any reason why this has to be in an after initialize a not a normal initializer?

Reply:
Great question!

Are you suggesting that apps including this gem should set up `Spring.watch` for packwerk within their own config/initializers file?

Considering the [available options](https://api.rubyonrails.org/classes/Rails/Railtie/Configuration.html), using after_initialize seems logical as it ensures Spring is loaded before this block is ran, provided it's defined.

Reply:
I think this should be either before_configuration or just a regular initializer